src/contracts/core/DelegationManager.sol

```diff
@@ -863,22 +854,29 @@ contract DelegationManager is
     function getDelegatableShares(
         address staker,
         IStrategy[] memory strategies
-    ) public view returns (uint256[] memory shares) {
+    ) public view returns (WithdrawableShares[] memory shares) {
         address operator = delegatedTo[staker];
         for (uint256 i = 0; i < strategies.length; ++i) {
             IShareManager shareManager = _getShareManager(strategies[i]);
             // TODO: batch call for strategyManager shares?
             // 1. read strategy deposit shares
-            shares[i] = shareManager.stakerStrategyShares(staker, strategies[i]);
+            
+            // forgefmt: disable-next-item
+            shares[i] = shareManager.stakerStrategyShares(staker, strategies[i])
+                .unwrap()
+                .wrapWithdrawable();
 
             // 2. if the staker is delegated, actual withdrawable shares can be different from what is stored
             // in the StrategyManager/EigenPodManager because they could have been slashed
             if (operator != address(0)) {
                 uint64 totalMagnitude = allocationManager.getTotalMagnitude(operator, strategies[i]);
-                shares[i] = WithdrawableShares.unwrap(
-                    Shares.wrap(shares[i]).toDelegatedShares(depositScalingFactors[staker][strategies[i]])
-                        .toWithdrawableShares(totalMagnitude)
-                );
+                
+                // forgefmt: disable-next-item
+                shares[i] = shares[i]
+                    .unwrap()
+                    .wrapShares()
+                    .toDelegatedShares(depositScalingFactors[staker][strategies[i]])
+                    .toWithdrawableShares(totalMagnitude);
```

Comment:
this is kind of confusing. probably better to not store the initial values in `shares[i]`